docs/css/homepage.css

```diff
@@ -0,0 +1,69 @@
+.homepage div[role="main"] {
+    width: 100%;
+}
+
+body.homepage div.container img {
+    border-width: 0;
+    margin: 0;
+    padding: 0;
+    background-color: transparent;
+}
+
+.homepage .top {
+    padding: 20px 0;
+    text-align: center;
+    color: #26282a;
+    width: 100%;
+    border-bottom: 1px solid #f1f1f5;
+}
+
+.homepage .top p {
+    font-size: 16px;
+    font-weight: 300;
+}
+
+.homepage .top img {
+    height: 62px;
+    width: 50px;
+    margin: 0;
+    padding: 0;
+}
+
+.homepage .top h1 {
+    color: #003abc;
+    font-size: 26px;
+    font-weight: 300px;
+    margin: 0;
+    margin-top: 10px;
+    padding: 0;
+}
+
+.homepage h2 {
+    color: #003abc;
+    margin: 0;
+    padding: 0;
+    font-size: 24px;
+    font-weight: 300;
+}
+
+.homepage .row {
+    padding: 20px 0;
+    margin: 0;
+    border-bottom: 1px solid #f1f1f5;
+}
+
+.homepage .row:last-child {
+    border-width: 0px;
+}
+
+.homepage .row p {
+    width: 100%;
+    font-weight: 300;
+    font-size: 16px;
+}
+
+/*@media (min-width: 768px) {
```

Comment:
It was making everything look funky in my laptop, but I'm not sure if the min-width just needs to be changed?